pwndbg/disasm/__init__.py

```diff
@@ -100,11 +100,13 @@ def get_one_instruction(address):
         pwndbg.disasm.arch.DisassemblyAssistant.enhance(ins)
         return ins
 
-def one(address=None):
+
+
+def one(address='unset_argument'):
+    if address == 'unset_argument':
```

Comment:
Where is `None` being explicitly passed in?  That sounds like a bug elsewhere that we're covering up with this statement.